cmd/libsnap-confine-private/experimental.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SNAP_CONFINE_EXPERIMENTAL_H
+#define SNAP_CONFINE_EXPERIMENTAL_H
+
+#include <stdbool.h>
+
+typedef enum sc_feature_flag {
+	SC_PER_USER_MOUNT_NAMESPACE,
+} sc_feature_flag;
+
+/**
+ * sc_experimental_flag_active returns true if a given experimental flag
+ * has been activated by the user via "snap set core experimental.xxx=true".
+ * This is determined by testing the presence of a file in /var/lib/snapd/features/
+ * that is named after the flag name.
+**/
+bool sc_experimental_flag_active(sc_feature_flag flag);
```

Comment:
I'd rename this to `sc_feature_flag_active`, per my general comment, and the file to `features.h` or some such. Current file name suggests it's experimental snap-confine stuff.